vulnerable_people_form/form_pages/shared/location_tier.py

```diff
@@ -6,19 +6,27 @@
 
 def update_is_postcode_in_england(postcode, app):
     postcode_in_england = is_postcode_in_england(postcode)
+
+    if postcode in app.config["POSTCODE_TIER_OVERRIDE"]:
+        postcode_in_england = True
+
     set_is_postcode_in_england(postcode_in_england)
 
 
 def update_location_tier_by_uprn(uprn, app):
-    if app.is_tiering_logic_enabled:
+    if uprn == 999:
```

Comment:
You could get the postcode from session here and do the logic on that. Which would enable us to set different tiers from uprns